app/scripts/check_requirements

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env php
+<?php
+
+// Script should not be called from the Web
+if (php_sapi_name() != 'cli') {
+    die('Nope');
+}
+
+/**
+ * Colorize the shell output
+ *
+ * @param string $text The text.
+ * @param string $color Color of the text: green, red or nothing.
+ * @return string
+ */
+function colorizeOutput($text, $color) {
+    switch ($color) {
+            case 'green':
+                // White on green
+                $output = "\033[1;37m\033[42m{$text}";
+                break;
+            case 'red':
+                // White on red
+                $output = "\033[1;37m\033[41m{$text}";
+                break;
+            default:
+                $output = $text;
+                break;
+        }
+        $output .= "\033[0m";
+        return $output;
+    }
+
+/**
+ * Check current configuration.
+ * 
+ * @param bool   $condition The type of alert.
+ * @param string $message   The message to display.
+ * @param string $help      Help about the message.
+ * @param string $suggested_fix A message that help to fix the issue.
+ *
+ * @return void
+ */
+function check($condition, $message, $help = '', $suggested_fix = '') {
+    $result = $condition ? colorizeOutput('OK', 'green') : colorizeOutput('FAILED', 'red');
+    $text = "* {$message}: {$result}\n";
+
+    if (! $condition) {
+        if(! empty($help)) { 
+            $text .=  "  {$help}\n";
+        }
+        if(! empty($suggested_fix)) {
+            $text .=  "  {$suggested_fix}\n";
+        }
+    }
+    
+    echo $text;
+}
+
+/**
+ * Check if the directory is empty
+ *
+ * @param string $dir The path.
+ * @return bool
+ */
+function isEmptyDir($dir) {
+    if(!file_exists($dir) || !is_dir($dir)) {
```

Comment:
When negating a condition, leave a space after !. I also believe that [is_dir](http://php.net/manual/en/function.is-dir.php) already returns false if the folder is missing
```
if(! is_dir($dir))
```